Source/WebCore/platform/graphics/ImageSource.cpp

```diff
@@ -64,7 +64,7 @@ ImageSource::ImageSource(Ref<NativeImage>&& nativeImage)
 ImageSource::~ImageSource()
 {
     ASSERT(!hasAsyncDecodingQueue());
-    ASSERT(&m_runLoop == &RunLoop::current());
+    ASSERT(m_runLoop.isCurrent());
```

Comment:
Alternatively this can store the runloop threadassertion as a member variable.

Reply:
I like `assertIsCurrent(m_runLoop);` better, no need to use `#if ASSERT_ENABLED`